openedx/core/djangolib/tests/utils.py

```diff
@@ -0,0 +1,61 @@
+from django.core.cache import caches
+from django.test import TestCase, override_settings
+
+
+class CacheSetupMixin(TestCase):
+    """
+    This class can be used to enable specific django caches for
+    specific the TestCase that it's mixed into.
+
+    Usage:
+
+    Use the ENABLED_CACHES to list the names of caches that should
+    be enabled in the context of this TestCase. These caches will
+    use a loc_mem_cache with the default settings.
+
+    Set the class variable CACHES to explicitly specify the cache settings
+    that should be overridden. This class will insert those values into
+    django.conf.settings, and will reset all named caches before each
+    test.
+
+
```

Comment:
Maybe a note here about what happens if you specify both, in case they get confused (e.g. they have a CACHES value that they define separately and share across multiple test cases, but each test case specifies a different ENABLED_CACHES).